tests/dummy/app/pods/components/frost-create-form/component.js

```diff
@@ -23,6 +26,12 @@ export default Component.extend({
     ]
   },
 
+  clearForm () {
+    this.set('userValue', {})
+  },
+  closeModal () {
+    this.get('remodal').close('my-awesome-modal')
```

Comment:
Same comment about promoting variables to hold the "id" so that we don't get disconnects